lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -201,7 +212,10 @@ class EngineAutofillForm {
       e.preventDefault();
     }));
 
-    _hideAutofillElements(formElement);
+    // We need to explicitly disable pointer events on the form in Safari Desktop,
+    // so that we don't have pointer event collisions if users hover over or click
+    // into the invisible autofill elements within the form.
+    _hideAutofillElements(formElement, shouldDisablePointerEvents: isSafariDesktopStrategy);
```

Comment:
It's not clear to me why we are not passing `shouldHideElement: !isSafariDesktopStrategy` here.

Reply:
For the autofill to work, we only need the actual `<input>` elements visible for Safari and don't have to worry about the `<form>`.

As for why we need to disable pointer events here on an "invisible" form, Safari has this weird behavior where the `<form>` element still registers as the target, even if the input child has disabled pointer events.  This doesn't occur on Chrome or Firefox from my testing.

Reply:
Oh nvm. I didn't notice that this is for `<form>` while the other one below is for `<input>` elements.